### DISCUSSION TITLE
Power Meter Settings - JSON - Results = difference between two values

### DISCUSSION THREAD

**Discussion by FilipeAmadeuO** [hoylabs/OpenDTU-OnBattery#1296] [Q&A] [answered]
Hello
I have configured Power Meter Settings - JSON correctly but i am only able to get one result. 
Is there any chance for the results to be the difference between two variables ?

Explaining better:
I get export and import energy value. The goal is to get the difference export minus import.

**Comment by FilipeAmadeuO** [comment, marked as answer]
Found solution by configurating Phase 2 + Change Sign.
Any way to change the text "Grid Power" in LiveView Page ?

**Comment by schlimmchen** [comment]
> Any way to change the text "Grid Power" in LiveView Page ?

To what any why? No, this is not possible without patching and rebuilding the web app.
